readme.md

```diff
@@ -198,124 +178,125 @@ Solar System
 Milky Way
 ```
 
-Absolute paths start with a `/` and go from top down (least specific to more specific):
+Absolute paths start with a `/` and go from the top down...
 
 ```bash
+# General Assembly in path form...
 /Milky_Way/Solar_System/Earth/USA/Washington_DC/1133_15th_St_NW/8th_Floor/GA
-# or a realistic example
+
+# ...but here's a more realistic example...
 /Users/adambray/code/work/general_assembly/courses/wdi/dc/dc8/lessons/cli-intro/
 ```
 
-The first slash essentially means "start at the root of the computer's file system".
+The first slash essentially means "start at the root of the computer's file system."
 
-Some absolute paths instead start with a `~`. This is a shortcut to the absolute
-path of our home directory. So the above absolute path could also be written as
+Some absolute paths instead start with a `~`. This is a shortcut to the absolute path of our home directory. So the above absolute path could also be written as
 
 ```
 ~/code/work/general_assembly/courses/wdi/dc/dc7/curriculum/01-front-end/
 ```
 
 #### Relative Paths
 
-Relative paths are interpreted starting from the directory we're in (aka the
-current working directory).
+Relative paths are interpreted as starting from the current working directory. They start with anything but a `/` or `~`.
 
-They start with anything but a slash `/` or a tilde `~`.
-
-So if we were in our home directory, the path to our work folder could be written
+<!-- AM: Are student folders already set up in this way? If not, need earlier instructions? -->
```

Comment:
I'll begin setting up the directory structure suggested at the end of the lesson: https://github.com/ga-wdi-lessons/cli-intro/tree/wdi10-update#directory-structure